celery/app/base.py

```diff
@@ -733,7 +733,8 @@ def send_task(self, name, args=None, kwargs=None, countdown=None,
             producer = amqp.Producer(connection, auto_declare=False)
         with self.producer_or_acquire(producer) as P:
             with P.connection._reraise_as_library_errors():
-                self.backend.on_task_call(P, task_id)
+                if not ignore_result:
+                    self.backend.on_task_call(P, task_id)
```

Comment:
Interesting. So this only affects backends that implement the `on_task_call` method?